userspace/libscap/engine/bpf/scap_bpf.c

```diff
@@ -1756,7 +1756,18 @@ const struct scap_stats_v2* scap_bpf_get_stats_v2(struct scap_engine_handle engi
 				}
 				stats[offset].type = STATS_VALUE_TYPE_U64;
 				stats[offset].flags = PPM_SCAP_STATS_LIBBPF_STATS;
-				strlcpy(stats[offset].name, info.name, STATS_NAME_MAX);
+				/* This could happen on old kernels where we don't have names inside the info struct
```

Comment:
@Andreagit97 on that note: Just occurred to me, perhaps we could add some more comments throughout that libbpf stats are only supported starting kernel 5.1 or if those feature were back ported. We check for the kernel settings plus in the scap open example I give hints, but maybe we can do more even more?

Reply:
In addition, could you clarify what "old kernels" here means?

Reply:
good point! Done :)